django/core/checks/compatibility/django_4_0.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from .. import Error, Tags, register
+
+
+@register(Tags.compatibility)
+def check_csrf_trusted_origins(app_configs, **kwargs):
+    for origin in settings.CSRF_TRUSTED_ORIGINS:
+        if '://' not in origin:
+            return [Error(
+                'As of Django 4.0, the values in the CSRF_TRUSTED_ORIGINS '
+                'setting must start with a scheme (usually http:// or '
+                'https://) but found %s. Read the release notes for details.'
+                % origin,
+                id='4_0.E001',
+            )]
+    return []
```

Comment:
I would collect errors for all origins
```suggestion
            errors.append(Error(
                'As of Django 4.0, the values in the CSRF_TRUSTED_ORIGINS '
                'setting must start with a scheme (usually http:// or '
                'https://) but found %s. Read the release notes for details.'
                % origin,
                id='4_0.E001',
            ))
    return errors
```